data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/GrokLogstashPluginAttributeMapperHelper.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.logstash.mapping;
+
+import org.opensearch.dataprepper.logstash.model.LogstashAttribute;
+
+import java.util.Map;
+
+public interface GrokLogstashPluginAttributeMapperHelper {
```

Comment:
These should be package protected. I'd also be happy to see them as static inner classes because their scope of use is so small.